api/ereqs_admin/migrations/0002_auto_20180116_2105.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-01-16 21:05
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    Theme = apps.get_model('admin_interface', 'Theme')
+    Theme.objects.create(
```

Comment:
This creates a second theme in the database. Shouldn't we be modifying the existing entry instead? That'd allow us to specify only the fields that are changing.